apps/projects/app/components/Content/IssueDetail/index.js

```diff
@@ -1 +1,60 @@
-export { default as IssueDetail } from './IssueDetail'
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  useLayout,
+} from '@aragon/ui'
+import EventsCard from './EventsCard'
+import DetailsCard from './DetailsCard'
+
+const IssueDetail = ({ issue }) => {
+  const { layoutName } = useLayout()
+  const columnView = layoutName === 'small' || layoutName === 'medium'
+
+  return columnView ? (
+    <div css="display: flex; flex-direction: column">
+      <div css={`
+          min-width: 330px;
+          width: 100%;
+          margin-bottom: ${layoutName === 'small' ? '0.2rem' : '2rem'};
+        `}
+      >
+        <DetailsCard issue={issue} />
+      </div>
+      <div css="min-width: 330px; width: 100%">
+        <EventsCard issue={issue} />
+      </div>
+    </div>
+  ) : (
+    <div css="display: flex; flex-direction: row">
+      <div css={`
+          max-width: 705px;
+          min-width: 350px;
+          width: 70%;
+          margin-right: 2rem;
+        `}
+      >
+        <DetailsCard issue={issue} />
+      </div>
+      <div css="flex-grow: 1">
+        <EventsCard issue={issue} />
+      </div>
+    </div>
+  )
+}
+  
+IssueDetail.propTypes = {
+  issue: PropTypes.shape({
```

Comment:
🚀🎉 Good! we should shape `issue` PropTypes on all the other places where it is being used like in this file, or even better, define its shape in an external file and import the type in all places where is being used!

This should apply to every proptype!

Reply:
One proptype at a time... :)